setBrightness.sh

```diff
@@ -0,0 +1,16 @@
+TYPE=$1
+LEVEL=$2
+
+echo "Setting wallpaper brightness..."
+
+WALLPAPER_PATH="$(pwd)/wallpaper.png"
+
+nice python adjustWallpaper.py $TYPE $LEVEL $WALLPAPER_PATH
```

Comment:
Could you change `python` to `python3` as users with default 2.x version can run into error.

Reply:
updated the pull request